inc/Classes/Func.php

```diff
@@ -1118,4 +1118,56 @@ public function isModActive($mod, &$caption = '')
 
         return array_key_exists($mod, $this->ActiveModules);
     }
+
+    /**
+     * Searches through a text and replaces occurences of %VARIABLENAME% with their counterpart.
+     * Just has basic stuff required to make information pages more dynamic, more to be added
+     * Be careful that you only expose uncritical commonly visible values or user-specifc information, otherwise this could be used to leak important data.
+     *
+     * @param string $text The text to replace placeholders in
+     *
+     * @global array $auth fetches userid for replacement
+     *
+     * @return string The text with placeholders replaced
+     */
+    public function replaceVariables($text) {
+        global $auth;
+
+        //initialize replacement array
+        $placeholderNames = [];
+        $replacementValues = [];
+
+            $placeholderNames []= '%TEST%';
+            $replacementValues []= 'Test OK!';
```

Comment:
Is %TEST% really needed?

Reply:
Nope. But it also does not hurt ;)
Fine to remove